### ISSUE TITLE
pseudocode

### ISSUE THREAD

**Issue by jdmar3** [inls718/codebase#19]
Look at this about planning and writing pseudocode. 

https://www.khanacademy.org/computing/computer-programming/programming/good-practices/p/planning-with-pseudo-code

**Comment by hAlbertK** [comment]
Will do!

On Tue, Mar 7, 2017 at 2:25 PM, John D. Martin III <notifications@github.com

**Comment by griffinac** [comment]
Hi @hAlbertK ! I saw something was pushed from the master for the decision tree pseudocode...is this the file we should be working in? :)

**Comment by hAlbertK** [comment]
yes it will be! we will work in decision-tree branch and merge our work
when we are done

**Comment by Aricela** [comment]
Oh, I created a pull request into master branch because I thought individual edit was supposed to a branch off the master, and get merged into master when it was done.

BtW I created a workflow_pseudocode file in the decision-tree branch. As of latest commit 8fc2d2b , it only covers the Welcome screen of one workflow. (IMG_4020 by fhouyang). So feel free to work on the other screens of this workflow, or start integrating others.

**Comment by AndiZ23** [comment]
Are we going to work on your workflow_pseudocode file, or create a new file separately?

**Comment by Aricela** [comment]
I invite all of you to work on mine. :O I think we need to merge them all eventually, so it might be easier to just start out all working on the same file.

**Comment by AndiZ23** [comment]
OK. I'm working on the workflow_pseudocode file. Then when committing the work, is that right to create a new branch and start a pull request?

**Comment by AndiZ23** [comment]
Hi all, just remind that we can find workflow sketches from repositories besides [sketches](https://github.com/inls718/sketches): [inls718/workflows](https://github.com/inls718/workflows) and [inls718/Interaction-IX,](https://github.com/inls718/Interaction-IX).
